tests/Umbraco.Tests.AcceptanceTest/package.json

```
@@ -8,7 +8,7 @@
     "test": "npx playwright test DefaultConfig",
     "all": "npx playwright test",
     "createTest": "node createTest.js",
-    "smokeTest": "npx playwright test DefaultConfig --grep \"@smoke\""
+    "smokeTest": "npx playwright test DataType"
```

```suggestion
    "smokeTest": "npx playwright test DefaultConfig --grep \"@smoke\""
```
After the tests have run on the pipeline, this should be committed